source/extensions/filters/http/cors/cors_filter.cc

```diff
@@ -114,6 +118,18 @@ bool CorsFilter::isOriginAllowed(const Http::HeaderString& origin) {
   return false;
 }
 
+bool CorsFilter::isOriginAllowedRegex(const Http::HeaderString& origin) {
+  if (allowOriginRegexes() == nullptr) {
+    return false;
+  }
+  for (const auto& regex : *allowOriginRegexes()) {
```

Comment:
Would have been great if `for()` just skips in case `allowOriginRegexes` is a `nullptr`, like in Go.